sync-config.sh

```diff
@@ -35,3 +36,22 @@ find /config -type f -iname "*.secret" -print0 \
         gpg2 --decrypt ${out_file}.gpg > ${out_file} 2> /dev/null
         rm ${out_file}.{gpg,secret}
 done
+
+find /config -type f -iname "*.env" -print0 \
+  | while IFS= read -r -d $'\0' file; do
+        while IFS='' read -r line || [[ -n "$line" ]]; do
+            if [[ "${line:0:1}" != "#" ]] && [[ "$line" == *"="* ]]; then
+                key=${line%%=*}
+                value=${line#*=}
+                decrypted_value=$(printf -- "$gpg_template" "$value" | gpg2 -d 2>/dev/null)
+                if [ "$?" -eq 0 ]; then
+                        echo $key=$decrypted_value >> ${file}.decrypted
```

Comment:
This is missing the prefix check like:
https://github.com/pebble/git-deploy-environment/blob/master/bin/get-app-environment.sh#L60

Not sure if dropping that pattern was intentional; the idea is to prevent swapping ciphertexts to expose secrets. i.e. I copy `SECRET_KEY=` to `VERSION=` and get the plaintext available at some `GET /version` endpoint.